File: Programming_Basics_with_Python/while_loop_lab/05._account_balance.py
total_sum = 0
command = input()
while command != "NoMoreMoney":
    sum = float(command)
    if sum < 0:
        print("Invalid operation!")
        break
    else:
        print(f"Increase: {sum:.2f}")
        total_sum += sum
    command = input()
print(f"Total: {total_sum:.2f}")

File: Programing_Basics_with_JS/whileLoopLab/05.AccountBalance.js
function accountBalance(input) {
    let index = 0;

    let data = input[index];
    index++;

    let totalSum = 0;

    while(data !== 'NoMoreMoney') {
        let sum = Number(data);

        if (sum < 0) {
            console.log('Invalid operation!');
            break;
        }
        totalSum += sum;
        console.log(`Increase: ${sum.toFixed(2)}`);
        data = input[index];
        index++;
    }

    console.log(`Total: ${totalSum.toFixed(2)}`);
}

accountBalance(["5.51", 
"69.42",
"100",
"NoMoreMoney"])


accountBalance(["120",
"45.55",
"-150"])
